src/main/java/tr/edu/ogu/ceng/dao/InternshipRegistryJpaRepositoryInterface.java

```diff
@@ -0,0 +1,9 @@
+package tr.edu.ogu.ceng.dao;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import tr.edu.ogu.ceng.model.InternshipRegistry;
+
+public interface InternshipRegistryJpaRepositoryInterface extends JpaRepository<InternshipRegistry, Long> {
```

Comment:
InternshipRegistryJpaRepositoryInterface  şeklinde uzunca bir isimdense 
InternshipRegistryRepository yeterli olacaktır.

Reply:
Aslında bu şekilde iki dosya oluşturmamın sebebi Cüneyt Hocamızın da kendi videolarında aynı yolu izlemiş olmasıydı. Adı kısa olan dosya doğrudan Repository sınıfını temsil ediyor, ismi uzun olan sınıf ise JPA repository'i extend etmek için var. Nasıl bir çözüm önerirsiniz hocam?

Reply:
@ykartal düzelttim hocam.